azure/leftovers.go

```diff
@@ -110,23 +108,34 @@ func NewLeftovers(logger logger, clientId, clientSecret, subscriptionId, tenantI
 		return Leftovers{}, errors.New("Missing tenant id.")
 	}
 
-	oauthConfig, err := adal.NewOAuthConfig(azurelib.PublicCloud.ActiveDirectoryEndpoint, tenantId)
-	if err != nil {
-		return Leftovers{}, fmt.Errorf("Creating oauth config: %s\n", err)
-	}
+	// oauthConfig, err := adal.NewOAuthConfig(azurelib.PublicCloud.ActiveDirectoryEndpoint, tenantId)
+	// if err != nil {
+	// 	return Leftovers{}, fmt.Errorf("Creating oauth config: %s\n", err)
+	// }
 
-	token, err := adal.NewServicePrincipalToken(*oauthConfig, clientId, clientSecret, azurelib.PublicCloud.ResourceManagerEndpoint)
+	// token, err := adal.NewServicePrincipalToken(*oauthConfig, clientId, clientSecret, azurelib.PublicCloud.ResourceManagerEndpoint)
+	// if err != nil {
+	// 	return Leftovers{}, fmt.Errorf("Creating service principal token: %s\n", err)
+	// }
```

Comment:
I think all the commented out sections can be removed now.

Reply:
done